.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: dart-lang/setup-dart@v1
 
       - name: Install Dependencies
-        run: dart pub get
+        run: flutter pub get
```

Comment:
Should this be fvm flutter?

Reply:
fvm is not installed on the project. It looks like the build workflows are totally broken so I will have to try and fix them, I might add it then, although I don't think flutter is actually needed here except for the examples